resources/js/Pages/Contacts/Index.jsx

```
@@ -0,0 +1,314 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { Box, FormControlLabel, Select, Switch } from "@mui/material";
+import { DataGrid, GridToolbar } from "@mui/x-data-grid";
+import { CheckCircle, Error } from "@mui/icons-material";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)